lib/Db/ConnectionInterface.php

```diff
@@ -297,4 +297,9 @@ public function escapeLike(string $like): string;
      * @return \PDO
      */
     public function getWrappedConnection();
+    
+    /**
+     * @return \Doctrine\DBAL\Schema\AbstractSchemaManager
+     */
+    public function getSchemaManager();
```

Comment:
cannot add that since it would be BC break ...

Reply:
@fashxp And when we add this via `@property` PhpDoc and make this as comment for Pimcore 11? Then everybody has the notice to add this method. An exception is thrown either way.

Reply:
I reverted the change as @fashxp  suggested. If you are not happy with it yet, then please just add your suggestion in another PR. By all means...this is just the Ecommerce installer and not a core component.

Reply:
@blankse just decided to add your suggestion. hope that I got it right...